openapi/components/schemas/Journal/JournalAccount.yaml

```
@@ -0,0 +1,22 @@
+type: object
+required:
+  - name
+properties:
+  id:
+    description: ID of the journal account.
+    readOnly: true
+    allOf:
+      - $ref: ../ResourceId.yaml
+  name:
+    type: string
+    description: Name of the journal account.
+    example: Unearned revenue
+  description:
+    type: string
+    nullable: true
+  createdTime:
+    description: Creation date/time.
```

issue: [Timestamp fields](https://github.com/Rebilly/api-definitions/blob/main/WRITING-STYLE.md#timestamp-fields)
```suggestion
    description: Date and time when the journal record was created.
```

---

Past time is prohibited :wink:

---

Thanks for spotting this 🙏